templates/revel/files/README.md

```diff
@@ -0,0 +1,35 @@
+# Revel template for Platform.sh
+
+This project provides a template application using the Revel web framework on Platform.sh. It is primarily an example, although could be used as the starting point for a real project.
+
+## Starting a new project
+
+To start a new project based on this template, follow these 3 simple steps:
+
+1. Clone this repository locally.  You may optionally remove the `origin` remote or remove the `.git` directory and re-init the project if you want a clean history.
+
+2. Create a new project through the Platform.sh user interface and select "Import an existing project" when prompted.
+
+3. Run the provided Git commands to add a Platform.sh remote and push the code to the Platform.sh repository.
+
+That's it!  You now have a working "hello world" level project you can build on.
+
+## Structure
+
+This project relies on Go module support in Go 1.11 and later.  You should commit your `go.mod` and `go.sum` files to Git, but not the `vendor` directory.
```

Comment:
Except it's not, it looks like.  It's doing it the old-school way, which... does it even work on Psh at all? :smile: I could only barely make GOPATH work on Psh.

Reply:
This one took me the longest by far to get running.

Reply:
This template also has the hardcoded httpport = 8888 in it, so that should be updated as well if you can suggest a fix in https://github.com/platformsh/template-builder/pull/88